tests/pillar/cluster.sls

```diff
@@ -4,6 +4,12 @@ horizon:
     version: liberty
     secret_key: secret
     session_timeout: 43200
+    plugin:
```

Comment:
that is the default supported platform. So probably we may have a need to modify the formula not to require theme. etc Is there any dashboard package available cross platforms?